src/software/ai/ai.h

```diff
@@ -41,6 +41,6 @@ class AI final
     std::shared_ptr<Navigator> getNavigator() const;
 
    private:
-    std::unique_ptr<HL> high_level;
     std::shared_ptr<Navigator> navigator;
+    std::unique_ptr<HL> high_level;
```

Comment:
Yeah, any changes where I reordered variable declarations are because of a bazel warning where the order of variable declarations don't match the order in which they are initialized in the .cpp implementation.

Reply:
Sweet, didn't know that 👍